doc/supported_file_formats.adoc

```diff
@@ -7,25 +7,22 @@ Minetest is able to read several types of file formats for use as assets in-game
 
 == Images
 [grid="all"]
-[options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^"]
+[options="header",stripes="even",cols="^.^,<.^,^.^,^.^"]
 |===
 | Format
 ^| General Usecase
 | Can be written by Lua
 | Can be read by Lua
-| Lossless optimization
 
 | PNG (`.png`)
 | All-round good losslessly compressed image format.
 | Yes (`minetest.encode_png()` ^(5.5+)^) (Third-party library ^(-5.4)^)
 | Yes (Third-party library)
-| http://optipng.sourceforge.net/[optipng] (use MTG's https://github.com/minetest/minetest_game/blob/master/utils/optimize_textures.sh[optimize_textures.sh] script for best result)
 
 | JPEG (`.jpg .jpeg`)
 | Lossily compressed images (e.g. photorealistic skyboxes and large textures)
 | No
 | No
-.3+| N/A
 
 | TGA (`.tga`)
 | Easy to programmatically encode and decode due to being a simple image format, and is to be preferred over BMP. Might be smaller than PNG at times due to a smaller file header.
```

Comment:
Compatibility note:

Minetest only supports TGA images of the following types:
* Type 1: uncompressed color-mapped (palette)
* Type 2: uncompressed RGB(A)
* Type 3: uncompressed black/white
* Type 10: RLE-compressed RGB(A)

If you see the error `Unsupported TGA file type`, a likely reason is that your image is a color-mapped (palette) RLE-compressed image of TGA type 9 and should be converted to a type 10 or type 2 image.

Reply:
Incomplete TGA support is definitely worth a warning.

Reply:
An unfortunate detail about this is that type 9 is in many cases the best mode for file size, as it combines two simple ways of compression that are already supported – color-maps (palettes) and RLE. Yet, adding support for type 9 makes little sense, as then texture creators would run the risk of textures not working in older versions of the game.